Automattic/jetpack
# Jetpack AI Assistant: Remove old deadcode from image modal

Fixes #

## Proposed changes:

* Removes dead image code
* Removes an unused flag

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






Only files used by the AI Assistant block are edited, so:

* Add an AI Assistant block
* Ask for some content
* Check that the toolbar options are removed while the response is streamed back
* Check that the toolbar options are shown again when the response is done
* Check that the block does not break
